build.sbt

```diff
@@ -38,7 +40,16 @@ def commonSettings = Seq(
   },
   pomIncludeRepository := { x => false },
   pomExtra := <inceptionYear>2013</inceptionYear>,
-  credentials ++= Util.loadCredentials()
+  credentials ++= Util.loadCredentials(),
+  pomPostProcess := { (node: XmlNode) =>
+    new RuleTransformer(new RewriteRule {
+      override def transform(node: XmlNode): XmlNodeSeq = node match {
+        case e: Elem if e.label == "dependency" && e.child.exists(child => child.label == "artifactId" && child.text.contains("testutil")) =>
+          Comment(s"Ommitted test-util dependency")
```

Comment:
Are you trying to avoid `test->test`? Why not move the test-util code into `Compile` configuration.